packages/loader/types/index.d.ts

```diff
@@ -1,8 +1,8 @@
 // TypeScript Version: 3.4
 
-import {ComponentType} from 'react'
-
+// tslint:disable-next-line: no-single-declare-module
 declare module '*.mdx' {
+  import {ComponentType} from 'react'
```

Comment:
out of curiosity, what is the benefit of moving the import inside the `declare module`?

Reply:
It is better to write declarations separately for each `module` IMO. Besides, the `disable` is not related.